src/Engine/Renderer/RenderObject/RenderObject.cpp

```diff
@@ -166,8 +166,8 @@ Core::Matrix4 RenderObject::getTransformAsMatrix() const {
     return getTransform().matrix();
 }
 
-Core::Aabb RenderObject::getAabb() const {
-    Core::Aabb aabb = m_mesh->getGeometry().computeAabb();
+Core::Aabb RenderObject::computeAabb() const {
+    const Core::Aabb& aabb = m_mesh->getGeometry().computeAabb();
```

Comment:
`computeAabb` returns a rvalue, you cannot use a reference here (at least, I don't understand what it does).
```suggestion
    Core::Aabb aabb = m_mesh->getGeometry().computeAabb();
```